packages/cyberstorm/src/components/Tooltip.tsx

```diff
@@ -0,0 +1,35 @@
+import React, { PropsWithChildren } from "react";
+import * as RadixTooltip from "@radix-ui/react-tooltip";
+import styles from "./componentStyles/Tooltip.module.css";
+
+export interface TooltipProps {
+  label?: string;
+}
+
+// Cyberstorm Tooltip component
+
+export const Tooltip: React.FC<PropsWithChildren<TooltipProps>> = ({
+  label,
+  children,
+}) => {
+  return (
+    <RadixTooltip.Provider delayDuration={150}>
+      <RadixTooltip.Root>
+        <RadixTooltip.Trigger>{children}</RadixTooltip.Trigger>
+        <RadixTooltip.Portal>
+          <RadixTooltip.Content
+            className={`${styles.tooltip}`}
+            sideOffset={5}
+            side={"right"}
+            collisionPadding={20}
+            sticky={"always"}
+            avoidCollisions={true}
+          >
+            {label}
```

Comment:
This is actually not necessary here since react doesn't render falsy values. `label ? label : null` is evaluated in the same way as just `label` but with a bit of extra steps.

Reply:
Oh, nice to know! so only need to do it when the value is wrapped in some element etc.👍